categories/Fruits.json

```diff
@@ -1,4 +1,5 @@
 {
     "apple": "I am in the hands of the rich.",
     "strawberry": "I am aromatic, bright, juicy and sweet."

```

Comment:
You have missed adding a comma (",") at the end here. As a result, when the program is executed, it will throw an invalid syntax error.